src/core/businessLogic/builders/categoriesBuilders.ts

```diff
@@ -0,0 +1,16 @@
+export class CategoryBuilder {
```

Comment:
Builders should be dedicated to complex object, in this case you just need a string so this should not needed

Reply:
For now, its, just a string, but the object will increase , because it's a category with subcategories, so whan=t do you think to add the news properties as needed?

Reply:
Sure, let's look forward and see how it evolves then